### DISCUSSION TITLE
Airflow Scheduler Keep Failing -  SSL error: sslv3 alert bad record mac

### DISCUSSION THREAD

**Discussion by tolkup** [apache/airflow#32130] [Q&A]
Hello can you help me with this error? 

I keep writing airflow scheduler -D but it keeps getting failing and giving heartbeat error . :( 

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1900, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/default.py", line 736, in do_execute
    cursor.execute(statement, parameters)
psycopg2.OperationalError: SSL error: sslv3 alert bad record mac

**Comment by potiuk** [comment]
Likely you have a problem with your postgres firewall/load balancing that keep on changing your host id when connecting to it over SSL. Nothing airflow specific, you need to look at your DB url/SSL connection and fix the problem. Similar issu here, but you will likly find more when you search:

https://stackoverflow.com/questions/4496941/curl-php-request-executes-50-of-the-time

**Reply by tolkup** [reply]
Thank for respond, with respect to your suggestion. 
I added sslmode=disable connection args to connection string. 

I hope it works :D

**Reply by potiuk** [reply]
Did it?

**Comment by insomnes** [comment]
Hey @tolkup ! Have you find the solution?
